src/routes/Send.tsx

```diff
@@ -636,6 +640,16 @@ export function Send() {
                         tags
                     );
 
+                    sentDetails.amount = amountSats();
+                    sentDetails.destination = address();
+                    sentDetails.txid = txid;
+                    sentDetails.fee_estimate = feeEstimate() ?? 0;
+                } else if (payjoinEnabled()) {
```

Comment:
I'm not sure a sweep should necessarily preclude payjoin on the `isMax` condition. A receiver offered a sweep could still take the opportunity to contribute an input or use transaction cut-through, and otherwise just broadcast the original psbt as a basic sweep. It doesn't break this PR but should be considered for incremental change later.